www/src/components/layout/content-container.js

```diff
@@ -0,0 +1,39 @@
+import React from "react"
+import PropTypes from "prop-types"
+import { css } from "react-emotion"
+
+import presets from "../../utils/presets"
+
+import { rhythm, options } from "../../utils/typography"
+
+const contentContainer = css`
+  background: red;
+  position: relative;
+`
+
+const ContentContainer = ({ children, hasSideBar = false }) => (
+  <div
+    className={contentContainer}
+    css={{
+      maxWidth: hasSideBar
+        ? rhythm(presets.maxWidthWithSidebar)
+        : rhythm(presets.maxWidth),
+      margin: `0 auto`,
+      padding: `${rhythm(1.5)} ${rhythm(options.blockMarginBottom)}`,
+      paddingBottom: rhythm(3.5),
+
+      [presets.Tablet]: {
+        paddingBottom: rhythm(1.5),
+      },
+    }}
```

Comment:
With the exception of `maxWidth`, it looks like all of this can move up into the initial style declarations.

Reply:
Actually, the content-container.js is unused now, it's job does now the `Content` component in `layout-default.js` I will remove the  file altogether. Sorry for the mistake.